### ISSUE TITLE
taskmanager did not invoke python beam by beam-python/flink-operator-custom-resource.yaml  

### ISSUE THREAD

**Issue by pzou1974** [lyft/flinkk8soperator#191]
https://github.com/lyft/flinkk8soperator/blob/master/examples/beam-python/flink-operator-custom-resource.yaml    I 

did kubectl create -f flink-operator-custom-resource.yaml   

but on the taskmgr pod log I did not see python example got invoked.

**Comment by tweise** [comment]
Hi, on the TM pod, you should see the Beam SDK worker process. Did Flink job start/terminate (check Flink UI)?

**Comment by pzou1974** [comment]
Yes. I checked UI both taskmgr and jobmgr are running. but failed to find completed python job.
Let me have one more test.
